_posts/2021-04-05-hhvm-4.104.markdown

```diff
@@ -0,0 +1,29 @@
+---
+title: "HHVM 4.104"
+layout: post
+author: jjergus
+category: blog
+---
+
+HHVM 4.104 is released! This release marks the end of support for 4.98;
+HHVM 4.99&ndash;4.103 remain supported, as do the 4.56, and 4.80 LTS releases.
+
+Note that HHVM 4.104 is not a long-term support (LTS) release as originally
+scheduled. Instead, HHVM 4.102 was
+[promoted to a LTS release](/blog/2021/03/29/extending-hhvm-4.102-support.html).
+
+# Highlights
+
+- Added support for
```

Comment:
These needs postponing, watchman/facts is not currently buildable. Some sources are missing from cmake, and at least macos's compiler doesn't like some of the files